http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/FollowupComponent.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import org.broadinstitute.dsde.workbench.leonardo.monitor.ClusterFollowupDetails
+
+import scala.concurrent.ExecutionContext
+import LeoProfile.api._
+
+case class FollowupRecord(clusterId: Long, status: String, masterMachineType: Option[String])
+
+class FollowupTable(tag: Tag) extends Table[FollowupRecord](tag, "CLUSTER_FOLLOWUP") {
+  def clusterId = column[Long]("clusterId", O.PrimaryKey)
+  def status = column[String]("status", O.Length(254))
+  def masterMachineType = column[Option[String]]("masterMachineType", O.Length(254))
+
+  def cluster = foreignKey("FK_CLUSTER_FOLLOWUP_CLUSTER_ID", clusterId, clusterQuery)(_.id)
+
+  def * =
+    (clusterId, status, masterMachineType) <> (FollowupRecord.tupled, FollowupRecord.unapply)
+}
+
+object followupQuery extends TableQuery(new FollowupTable(_)) {
+  def save(followupDetails: ClusterFollowupDetails, masterMachineType: Option[String]): DBIO[Int] =
+    followupQuery += FollowupRecord(followupDetails.clusterId,
+                                    followupDetails.clusterStatus.entryName,
+                                    masterMachineType)
+
+  def delete(followupDetails: ClusterFollowupDetails): DBIO[Int] =
+    baseFollowupQuery(followupDetails).delete
+
+  //TODO: should this differentiate itself between no cluster with that ID found, and no machine config found?
```

Comment:
I plan to clean this up before the PR is merged, but this TODO is more of a question to reviewers than myself. I think the answer is no, because to the pubsub reader which retrieves this, it makes no difference why there is no follow-up details, both result in the same no-op.

Reply:
I think returning an `Option` is fine, we don't need to differentiate… and the database should maintain referential integrity for the foreign key, so there should always be a corresponding CLUSTER defined.